src/UI/Index/Keybindings.hs

```diff
@@ -39,6 +39,7 @@ browseThreadsKeybindings =
     , Keybinding (V.EvKey V.KUp []) (listUp `chain` continue)
     , Keybinding (V.EvKey (V.KChar 'G') []) (listJumpToEnd `chain` continue)
     , Keybinding (V.EvKey (V.KChar '1') []) (listJumpToStart `chain` continue)
+    , Keybinding (V.EvKey (V.KChar '*') []) (toggleListItem `chain` listDown `chain` continue)
```

Comment:
So happy our UI is configurable enough that I can easily override this behaviour to *not* `listDown` :)

Reply:
Yeah I simply took the default mutt behaviour, but I also realised that there is some value by keeping the cursor state. Also by keeping the list items toggled, after tagging is applied (Keybinding change needed too).